services/api/services.api

```diff
@@ -52,6 +52,18 @@ public abstract interface class io/opentelemetry/android/internal/services/netwo
 	public abstract fun onNetworkChange (Lio/opentelemetry/android/common/internal/features/networkattributes/data/CurrentNetwork;)V
 }
 
+public final class io/opentelemetry/android/internal/services/network/NetworkUtils {
+	public static final field INSTANCE Lio/opentelemetry/android/internal/services/network/NetworkUtils;
+	public final fun getNetworkTypeName (I)Ljava/lang/String;
+	public final fun hasReadPhoneStatePermission (Landroid/content/Context;)Z
+	public final fun isValidString (Ljava/lang/CharSequence;)Z
+}
+
+public final class io/opentelemetry/android/internal/services/network/SubTypeFinder {
+	public fun <init> (Landroid/content/Context;Landroid/telephony/TelephonyManager;)V
+	public final fun get ()Ljava/lang/String;
+}
```

Comment:
Unless these need to form part of OpenTelemetry's public API, these symbols should be made non-public